crates/biome_cli/src/execute/migrate/eslint_any_rule_to_biome.rs

```diff
@@ -513,6 +523,11 @@ pub(crate) fn migrate_eslint_any_rule(
             let rule = group.use_iframe_title.get_or_insert(Default::default());
             rule.set_level(rule_severity.into());
         }
+        "jsx-a11y/img-redundant-alt" => {
```

Comment:
@Conaclos just so you know, I've updated the branch with `main` and ran `just gen-lint` and this change appeared, but it's probably ok and related to #2609, right?

Reply:
Yes. I forgot to generate the code. Thanks!